cloudfoundry-client-lib/src/main/java/org/cloudfoundry/client/lib/CloudControllerClient.java

```diff
@@ -46,6 +46,7 @@
 import org.cloudfoundry.client.lib.domain.InstancesInfo;
 import org.cloudfoundry.client.lib.domain.Staging;
 import org.cloudfoundry.client.lib.domain.Upload;
+import org.cloudfoundry.client.v2.serviceinstances.LastOperation;
```

Comment:
Not sure whether this is a good idea. So far we have a pattern of hiding the fact that we're using the reactor-cf-java-client. At least I don't remember a place where multiapps-controller uses classes from it directly.